charts/edge/README.md

```diff
@@ -31,7 +31,8 @@ helm install edge mezmo/edge \
 
 | **Key**                      | **Type** | **Default**      | **Description**
 | ---------------------------- | -------- | ---------------- | ----------------------------------------------------------------------
-| mezmoApiAccessToken          | string   |                  | Your Mezmo API access token
+| mezmoApiAccessToken       | string   |                 | Your Mezmo API access token. If `mezmoApiAccessSecret` is also specified it take precedence and this will be unused
```

Comment:
nit:
```
If `mezmoApiAccessSecret` is also specified it take precedence and this will be unused
```

Besides missing an "s" after "take" and a comma or two, I find this sentence kind of hard to follow. I think changing to something like

```
This has a lower precedence than `mezmoApiAccessSecret`
```

Gets the same idea across and is much simpler to grok.

Reply:
That is better, thanks. Updated!